corehq/form_processor/management/commands/run_sql.py

```diff
@@ -124,56 +124,12 @@ def run(sql, dbname):
     END || blob_id
 )::varchar(255)))
 """
-DROP_BLOBMETA_KEY = "DROP INDEX form_processor_xformattachmentsql_blobmeta_key"
-BLOBMETA_VIEW = """
-CREATE OR REPLACE VIEW blobs_blobmeta AS
-SELECT
-    "id",
-    "domain",
-    "parent_id",
-    "name",
-    "key",
-    "type_code",
-    "content_type",
-    "properties",
-    "created_on",
-    "expires_on",
-    "content_length"
-FROM blobs_blobmeta_tbl
-
-UNION ALL
-
-SELECT
-    -att."id" AS "id",
-    xform."domain",
-    att.form_id AS parent_id,
-    att."name",
-    (CASE
-        WHEN att.blob_bucket = '' THEN '' -- empty bucket -> blob_id is the key
-        ELSE COALESCE(
-            att.blob_bucket,
-            'form/' || REPLACE(att.attachment_id::text, '-', '')
-        ) || '/'
-    END || att.blob_id)::VARCHAR(255) AS "key",
-    CASE
-        WHEN att."name" = 'form.xml' THEN 2 -- corehq.blobs.CODES.form_xml
-        ELSE 3 -- corehq.blobs.CODES.form_attachment
-    END::SMALLINT AS type_code,
-    att.content_type,
-    att.properties,
-    xform.received_on AS created_on,
-    NULL AS expires_on,
-    att.content_length
-FROM form_processor_xformattachmentsql att
-    INNER JOIN form_processor_xforminstancesql xform
-        ON xform.form_id = att.form_id;
-"""
```

Comment:
nit: removing these doesn't fit with the commit message

Reply:
good point. I should have done that first in a separate commit. I think I'm going to ignore it.